README.md

```diff
@@ -132,6 +133,11 @@ FMPhotoEditor provides some default crops that will be fit to you.
 Type: `[FMCroppable]?` 
 Default: all crops provided by FMPhotoPicker.
 
+- <a name="ref-use-crop-first"></a>`useCropFirst`    
+An option that indicates which Crop or Filter view should be displayed first in `FMImageEditorViewController`.
```

Comment:
> An option that indicates which Crop or Filter view should be displayed first in `FMImageEditorViewController`.

Should be: An option that indicates whether the crop menu should be selected by default in the `FMImageEditorViewController`.